tests/system/test_root_metronome.py

```diff
@@ -240,7 +239,8 @@ def test_docker_job():
         assert len(client.get_runs(job_id)) == 1
 
 
-@shakedown.masters(1)
```

Comment:
we should probably also rename those annotations @kensipe to reflect the logic e.g.  `@shakedown.master_bigger_than` (which is not a great name)

Reply:
I am afraid that's a backward incompatible change :( if marathon is the only one using shakedown then it's probable possible...